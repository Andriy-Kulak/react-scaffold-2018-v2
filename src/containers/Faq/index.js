import React from 'react';
import Layout from '../../components/layout';

const Faq = () => (
  <Layout>
    <div>
      <h1>
        FAQ PAGE
      </h1>
    </div>
  </Layout>
);

export default Faq;
