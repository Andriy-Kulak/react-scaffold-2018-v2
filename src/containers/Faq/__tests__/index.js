/* eslint-env node, jest */

import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Adapter from 'enzyme-adapter-react-16';

import Faq from '../index';

Enzyme.configure({ adapter: new Adapter() });

describe('>>> Faq Standard Render Container ~~ Snapshot ~~', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(<Faq />);
    expect(toJson(renderedValue)).toMatchSnapshot();
  });
});
