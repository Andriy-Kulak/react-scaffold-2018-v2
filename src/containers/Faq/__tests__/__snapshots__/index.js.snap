// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Faq Standard Render Container ~~ Snapshot ~~ +++capturing Snapshot with 1`] = `
<Layout>
  <div>
    <h1>
      FAQ PAGE
    </h1>
  </div>
</Layout>
`;
